CDCgov/prime-simplereport
# Reduce params organization mutation part 2/3

# FRONTEND PULL REQUEST

## Related Issue

- Partially implements #5682

## Changes Proposed

- Update the mutations used in the `FacilityFormContainer` component.


## Additional Information
As part of this PR I also moved some other gql code to the operations file to follow the graphql standards of the project.

## Testing
Verify below functionality work as expected. This PR is available in dev3. 
- Add a new facility
- Update a facility
